modules/installation-configure-proxy.adoc

```diff
@@ -53,7 +53,9 @@ endif::bare-metal[]
 +
 [NOTE]
 ====
-The Proxy object's `status.noProxy` field is populated by default with the instance metadata endpoint (`169.254.169.254`) and with the values of the `networking.machineNetwork[].cidr`, `networking.clusterNetwork[].cidr`, and `networking.serviceNetwork[]` fields from your installation configuration.
+The Proxy object `status.noProxy` field is populated with the values of the `networking.machineNetwork[].cidr`, `networking.clusterNetwork[].cidr`, and `networking.serviceNetwork[]` fields from your installation configuration.
+
+For installations on Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure, and {rh-openstack-first}, the `Proxy` object `status.noProxy` field is also populated with the instance metadata endpoint (`169.254.169.254`).
```

Comment:
> the `Proxy` object 

For some reason, "Proxy" isn't showing up as monospaced in the preview output. If it looks right in your local build, maybe the preview is acting up?

Reply:
@jeana-redhat, oh, that's my fault because the preview doesn't update very quickly; I fixed this right before asking for a review.